### PR TITLE
Removed unused converters and renamed existing converter

### PR DESCRIPTION
**Pull request by nassaleh**
Issue: #
https://nventive.visualstudio.com/Umbrella/_workitems/edit/128561

## PR Type
What kind of change does this PR introduce?
Refactoring (no functional changes, no api changes)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes

### CODE REVIEW

File: src/SourceGenerators/System.Xaml.Tests/Test/XmlFiles/Generic.Native.xaml [hunk 1]
```diff
@@ -161,7 +161,7 @@
     </android:Style>
     <xamarin:Style BasedOn="{StaticResource NativeDefaultSlider}" TargetType="Slider" />
 
-    <converters:FromNullableBoolToReverseBoolConverter x:Key="nativeDefaultProgressBarReverseBool" />
+    <converters:UnoNativeDefaultProgressBarConverter x:Key="nativeDefaultProgressBarReverseBool" />
```
Comment (MatFillion) [inline src/SourceGenerators/System.Xaml.Tests/Test/XmlFiles/Generic.Native.xaml]: UnoNativeDefaultProgressBarReverseBoolConverter

File: src/SourceGenerators/System.Xaml.Tests.MS/XmlFiles/Generic.Native.xaml [hunk 2]
```diff
@@ -161,7 +161,7 @@
     </android:Style>
     <xamarin:Style BasedOn="{StaticResource NativeDefaultSlider}" TargetType="Slider" />
 
-    <converters:FromNullableBoolToReverseBoolConverter x:Key="nativeDefaultProgressBarReverseBool" />
+    <converters:UnoNativeDefaultProgressBarConverter x:Key="nativeDefaultProgressBarReverseBool" />
```
Comment (MatFillion) [inline src/SourceGenerators/System.Xaml.Tests.MS/XmlFiles/Generic.Native.xaml]: UnoNativeDefaultProgressBarReverseBoolConverter

File: src/Uno.UI/UI/UnoNativeDefaultProgressBarConverter.cs [hunk 3]
```diff
@@ -7,13 +7,9 @@
 namespace Uno.UI.Converters
 {
 	/// <summary>
-	/// This converter will return the opposite value of a nullable bool.
-	/// 
-	/// If null or false, the converter will return true.
-	/// 
-	/// This may be used to have the a reversed value bound to that of a ToggleButton.
+	/// Do not use outside of UNO
```
Comment (MatFillion) [inline src/Uno.UI/UI/UnoNativeDefaultProgressBarConverter.cs]: Uno

File: src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs [hunk 4]
```diff
@@ -33,18 +32,9 @@ public static partial class GenericStyles
 			InitPage();
 			InitWebView();
 			InitDatePicker();
-			InitFlipViewItem();
 			InitToggleSwitch();
 			InitFlyoutPresenter();
 
-#if !IS_UNO
```
Comment (MatFillion) [inline src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs]: why so many deletes?

Comment (nassaleh) [reply src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs]: @jeremiethibeault These were the changes you had asked me to do in my initial Pull Request that was part of VSTS. I assume these changes are still useful?

File: src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs [hunk 5]
```diff
@@ -278,29 +195,6 @@ private static void InitToggleSwitch()
 			Style.RegisterDefaultStyleForType(typeof(Windows.UI.Xaml.Controls.ToggleSwitch), style);
 		}
 
-		private static void InitContentControl()
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs]: Don't remove this. It's there for reference.

**Comment by MatFillion** [comment]
clean up commit history

File: src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs (hunk 4 above)
Comment (davidjohnoliver) [reply src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs]: @MatFillion This is all dead code referring to Umbrella controls (apart from InitContentControl - I don't know what's the deal with that)

Comment (jeremiethibeault) [reply src/Uno.UI/UI/Xaml/Style/Generic/GenericStyles.Android.cs]: Mmmm I must have been misunderstood in the past! No idea what we are talking about. :)

**Pull request merged**